test/e2e/custom_wait_rules_test.go

```diff
@@ -116,6 +116,7 @@ status:
 
 		require.Contains(t, res, "Current state as Failed")
 
-		require.Contains(t, err.Error(), "kapp: Error: waiting on reconcile crontab/my-new-cron-object-1")
+		require.Contains(t, err.Error(), "kapp: Error: Finished unsuccessfully")
+		require.Contains(t, err.Error(), "crontab/my-new-cron-object-1 (stable.example.com/v1) namespace: kapp-test (Current state as Failed)")
```

Comment:
For timeout, does it make sense to show as timed-out error instead of `(Current state as Failed)`.

Reply:
Sure. 
(except this is not a timed-out error)